docs/docs/10.1-animation.md

```diff
@@ -247,3 +247,31 @@ Any additional, user-defined, properties will become properties of the rendered
   ...
 </ReactTransitionGroup>
 ```
+
+### Using a Custom Render Function
+
+Rather than just specifying a different component wrapper with the `component` prop, you can also specify a custom render function with the `render` prop. This function takes an array of children as an argument.
+
+To render only the first child you could do something like:
+
+```javascript{1}
+function renderFirstChildOnly(children) {
+  return children[0];
```

Comment:
Note: this is actually not the case. If there is only a single child passed from JSX explicitly, you will get this child as a single element instead. The only correct way to manipulate children is to treat it as opaque data structure and use `React.Children`. In this case, to enforce a single child:

```js
function renderFirstChildOnly(children) {
  return React.Children.toArray(children)[0];
}
```

Ideally just `React.Children.onlyChild` should work but unfortunately it doesn’t flatten arrays right now. It’s a separate valid issue and we might need to implement this.. I think.

. This wouldn’t actually convert an array with a single element to an element (this is